ext/tags.go

```diff
@@ -22,6 +22,9 @@ var (
 
 	// PeerPort records port number of the peer
 	PeerPort = uint16Tag("peer.port")
+
+	// SamplingPriority determines the priority of sampling this Span.
+	SamplingPriority = uint32Tag("sampling.priority")
```

Comment:
No reason, I just took what's here - I don't fully understand this code. What's the point of these internal tag types which are all strings? Using this tag from `standardtracer` is now an awkward `SetTag(string(ext.SamplingPriority), ...)`.